rootfs/etc/nginx/lua/test/configuration_test.lua

```diff
@@ -0,0 +1,66 @@
+package.path = "./rootfs/etc/nginx/lua/?.lua;./rootfs/etc/nginx/lua/test/mocks/?.lua;" .. package.path
+_G._TEST = true
+
+local _ngx = {
+  shared = {
+      configuration_data = {
+        get = function(self, rsc) return {backend = true} end,
+        set = function(self, a) end,
+      }
+  },
+  print = function(msg) return msg end,
+  log = function() end,
+  var = {},
+  req = {
+      read_body = function() end,
+      get_body_data = function() return {body_data = true} end,
+  }
+}
+_G.ngx = _ngx
+
+local configuration = require("configuration")
+
+describe("Configuration", function()
+    context("Request method is neither GET nor POST", function()
+        it("should log 'Only POST and GET requests are allowed!'", function()
+            ngx.var.request_method = "PUT"
+            local s = spy.on(ngx, "print")
+            assert.has_no.errors(function() configuration.call() end)
```

Comment:
Nit: Just syntax, but I would write this as `assert.has_no.errors(configuration.call)`